infrastructure/staking-mechanics/offenses-and-slashes.md

```diff
@@ -1,4 +1,149 @@
+|           ForInvalid Vote            |           -            |                No                |        Yes (Mid Priority)         |                     No                      |
+|          AgainstValid Vote           |           -            |                No                |        Yes (Low Priority)         |                     No                      |
+| GRANDPA / BABE / BEEFY Equivocations |       0.01-100%        |               Yes                |                No                 |                     No                      |
+|    Seconded + Valid Equivocation     |           -            |                No                |                No                 |                     No                      |
+|     Double Seconded Equivocation     |           -            |                No                |                No                 |                     Yes                     |
+
+## Offenses
+
+!!! tip
+    To better understand the terminology used for offenses, getting familiar with the [parachain protocol](https://wiki.polkadot.network/docs/learn-parachains-protocol#parachain-protocol){target=\_blank} is recommended.
+
+On Polkadot, there are six main validator offenses, as shown below:
+
+- **Backing invalid** - a para-validator backs an invalid block
+- **`ForInvalid` vote** - a validator (secondary checker) votes in favor of an invalid block
+- **`AgainstValid` vote** - a validator (secondary checker) votes against a valid block, wasting network resources
+- **Equivocation** - a validator produces two or more identical blocks or votes
+- **GRANDPA and BEEFY equivocation** - a validator signs two or more votes in the same round on different chains
+- **BABE equivocation** - a validator produces two or more blocks on the relay chain in the same time slot
+- **Double seconded equivocation** - in a backing group of five para-validators, five backed parablocks are possible. Each parablock requires one seconded vote and at least two valid votes from the five potential backers, establishing an upper limit on the number of parablocks the system can manage while providing some flexibility for relay chain block authors. Backers must choose one parablock to second; seconding more than one incurs punishment. [Asynchronous backing](https://wiki.polkadot.network/docs/learn-async-backing){target=\_blank} complicates this by allowing backers to support blocks "into the future" optimistically rather than being restricted to one candidate per relay chain block. See the documentation on [Seconding Limit](https://paritytech.github.io/polkadot-sdk/book/node/backing/statement-distribution.html#seconding-limit){target=\_blank} for more information
+- **Seconded and valid equivocation** - a malicious node seconds a vote, taking absolute responsibility for it, and then falsely claims it is correct after someone else has taken responsibility. Once conflicting votes are detected by the system, the offense is reported
```

Comment:
I'm not in love with this because I think `ForInvalid` and `AgainstValid` are very specific things in the code, but I'm not sure about the other items in this list. I don't know enough here and tried looking into it a little bit but opted out of going down the rabbit hole haha, so I'll defer to @polkadot-developers/docs-content-team on this 🙂

Reply:
Also, I added some formatting to this list, but not the table. This will ideally in the future be formatted the same across the board, but I want to better understand the specifics of the other items first

Reply:
So from my knowledge (which is not a lot in this specific area, @filippoweb3, want to give this a shot?), these do all have references in code, whether it is an enum or a pallet specific function which specifies this offense (as is the case with BABE)

You are correct in that they references specific things, see here: https://paritytech.github.io/polkadot-sdk/master/cumulus_primitives_core/relay_chain/v8/slashing/enum.SlashingOffenceKind.html#

Perhaps we could rephrase them, like: "`ForInvalid`..." -> "Invalid vote"... to match the rest, or we could rename all of them, not sure which is better

Reply:
@Overkillus might have some insights here. We worked on the Wiki page about offenses this year. Unsure if a duplicate is needed on the devs docs. You could just refer to the Wiki page.

Reply:
@filippoweb3  - Ideally we move everything that is dev oriented (including Infra) from the Wiki to the Devs docs.

I would love to connect with him via Telegram as well so that we can speed this up 😃

Reply:
I did some work on this page to change a bit how it's laid out and how the information is presented. I'd love to have both teams take a look and let me know what you think!